### PR TITLE
Weather app - Rebeka Kovačič

### PR DESCRIPTION
**Pull request by rebkova**
Link to my deployed project: https://whatsup-weather.netlify.app/

So far my favourite project. 
Still, would love to spend less time with styling issues and more writing JS code!

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@800&display=swap" rel="stylesheet" />
+    <title>Rebkova weather app</title>
+  </head>
+  <body>
+    <section class="daily-weather">
+      <div class="time-location-date">
+        <p class="current-time" id="time">current time ...</p>
+        <p class="current-location" id="cityCountry">city ...</p>
```
Comment (MayaBlixt) [inline code/index.html:15]: Nice feature to display the current time and current date 👍  😃

File: code/script.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    //city + country
+    cityCountry.innerHTML = `${weatherObject.name}, ${weatherObject.sys.country}`;
+
+    //temperature
+    temp.innerHTML = `${weatherObject.main.temp.toFixed()}°`;
+
+    //description
+    description.innerHTML = weatherObject.weather[0].description;
+
+    //formatting unix stamps to time
```
Comment (MayaBlixt) [inline code/script.js:49]: Good comments in the script.js file 😄

**Review by MayaBlixt — APPROVED** [review]
Great job Rebeka! Love that you choose to use Lund for this app, the clean design and nice javascript code. Keep up the good work 😄  👍

File: code/script.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+const sunset = document.getElementById("sunsetTime");
+
+//formatting sunrise/sunset unix stamps (in ms)
+const formatUnix = (unixStamp) => {
+  const stampDate = new Date(unixStamp * 1000);
+  const stampTime = stampDate.toLocaleTimeString("se-SE", {
+    timeStyle: "short",
+  });
+  return stampTime;
+};
```
Comment (HIPPIEKICK) [inline code/script.js:18]: ⭐

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    sunset.innerHTML = sunsetTime;
+
+    document.getElementById("date").innerHTML = dateFormatted;
+    document.getElementById("time").innerHTML = timeFormatted;
+  });
+
+//FORECAST
+const API_URL_FORECAST = `https://api.openweathermap.org/data/2.5/forecast?q=Lund,Sweden&units=metric&APPID=${API_KEY}`;
+
+const forecastText = document.getElementById("5DayForecast");
```
Comment (HIPPIEKICK) [inline code/script.js:66]: JavaScript variables should start with a letter

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+      const weekday = date.toLocaleDateString("se-SE", { weekday: "short" });
+      const dailyTemp = day.main.temp.toFixed();
+
+      forecastText.innerHTML += `
+    <div class="daily-forecast">
+    <p class="weekday">${weekday.toLocaleLowerCase()}</p>
+    <p class="weekday-temp">${dailyTemp}°</p>
+    </div>`;
+    });
+  });
```
Comment (HIPPIEKICK) [inline code/script.js:93]: Indentation is a little off

File: code/style.css [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+}
+
+  span {
+    font-size: 30px;
+    font-weight: 500;
+  }
+
+  .daily-weather {
+    margin: 0 25px;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:18]: Indentation again ;)

**Review by HIPPIEKICK — APPROVED** [review]
Keep up the good work Rebeka! Nice design ;)

**Pull request closed without merging**